### PR TITLE
Use `#[udigest(as = ...)]` attribute

### PR DESCRIPTION
**Pull request by survived**

### CODE REVIEW

**Comment by maurges** [comment]
Looks good, but also isn't it unnecessary to change this

**Comment by survived** [comment]
I don't think it's unnecessary

File: Cargo.toml [hunk 1]
```diff
@@ -50,3 +50,10 @@ required-features = ["serde"]
 [package.metadata.docs.rs]
 all-features = true
 
+[patch.crates-io.udigest]
+git = "https://github.com/dfns/udigest"
+branch = "udigest_as"
+
+[patch.crates-io.udigest-derive]
+git = "https://github.com/dfns/udigest"
+branch = "udigest_as"
```
Comment (maurges) [inline Cargo.toml]: Shall we update those?

Comment (survived) [reply Cargo.toml]: Done

Comment (survived) [reply Cargo.toml]: I was waiting for generic-ec to be merged first, but only now realized that it's not patched here anyway

**Review by maurges — APPROVED** [review]

**Pull request merged**